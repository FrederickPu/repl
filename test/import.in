{"cmd": "import Lean"}

{"cmd": "def f := 42", "env": 0}

