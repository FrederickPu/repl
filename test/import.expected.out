{"results": [{"source": "import Lean\n  \n  ", "env": 0}]}

{"results": [{"source": "def f :=\n  42", "env": 1}]}

